cmd/csi-driver/Dockerfile

```diff
@@ -1,17 +1,45 @@
-FROM debian:stretch
-RUN apt-get update && \
-    apt-get install -y \
-    multipath-tools \
-    e2fsprogs \
-    btrfs-progs \
-    xfsprogs \
-    iproute2 \
-    util-linux \
-    systemd \
-    libisns0 \
-    gawk \
-    nfs-common \
-    dmidecode
+FROM registry.access.redhat.com/ubi7/ubi-minimal:7.7-138
+
+LABEL name="HPE CSI Driver for Kubernetes" \
+      maintainer="HPE Storage" \
+      vendor="HPE" \
+      version="1.1.0" \
+      summary="HPE CSI Driver for Kubernetes" \
+      description="The HPE CSI Driver for Kubernetes enables container orchestrators, such as Kubernetes and OpenShift, to manage the life-cycle of persistent storage." \
+      io.k8s.display-name="HPE CSI Driver for Kubernetes" \
+      io.k8s.description="The HPE CSI Driver for Kubernetes enables container orchestrators, such as Kubernetes and OpenShift, to manage the life-cycle of persistent storage." \
+      io.openshift.tags=hpe,csi,hpe-csi-driver
+
+WORKDIR /root
+COPY /LICENSE /licenses/
+
+RUN mkdir /chroot
+ADD chroot-host-wrapper.sh /chroot
+RUN chmod 777 /chroot/chroot-host-wrapper.sh
+RUN    ln -s /chroot/chroot-host-wrapper.sh /chroot/blkid \
```

Comment:
few other command I found in common-host-libs/linux 
udevadm
lsb_release
uname

Reply:
these are already part of UBI and works well within container as we mount /etc/os-release file paths. Udevadm is not yet used with CSI.